canonical-web-and-design/maas-ui
# Add page frame and settings navigation

Done:
- Add basic page layout. Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1330.
- Added basic settings nav.

QA:
- Load the UI. Notice the new header.
- Click on Settings.
- The header should change and you should get nav down the left (see https://app.zeplin.io/project/5ce5423a25863c1e0d7abb27/screen/5d1cbd18160fa1aff58546d0).
- Navigating to other settings pages (not General) should retain the heading/nav and highlight the page you're on.
